featuretools/entityset/entityset.py

```diff
@@ -566,12 +566,16 @@ def normalize_entity(self, base_entity_id, new_entity_id, index,
         for v in additional_variables + copy_variables:
             if v == index:
                 raise ValueError("Not copying {} as both index and variable".format(v))
-                break
+
+        for v in additional_variables:
+            if v == base_entity.time_index:
+                raise ValueError("Not moving {} as it is the base time index variable.".format(v))
```

Comment:
Let's suggest ways the user can resolve the error, either by setting `make_time_index` to `True` so it copies the base entity's time index by default, or by including the variable in `copy_variables`